test/parallel/test-domain-stack-empty-in-process-uncaughtexception.js

```diff
@@ -6,15 +6,18 @@ const assert = require('assert');
 
 const d = domain.create();
 
-process.on('uncaughtException', common.mustCall(function onUncaught() {
+process.once('uncaughtException', common.mustCall(function onUncaught() {
   assert.strictEqual(
     process.domain, null,
-    'domains stack should be empty in uncaughtException handler');
+    `Domains stack should be empty in uncaughtException handler
+     but the value of process.domain is ${JSON.stringify(process.domain)}`);
```

Comment:
This will print the message on two lines and add 4 spaces at the beginning of the second one. I think it's better to use a single line.

Reply:
If a single line of code exceeds 80 chars, you can keep it as a single line of output like this:

```js
'Domains stack should be empty in uncaughtException handler' +
`but the value of process.domain is ${JSON.stringify(process.domain)}`);
```

Reply:
Alternatively, you can simply remove the `message` (3rd) argument entirely and the default behavior will print the value of `process.domain`. If you think the contents of the message are valuable, you can put them as a comment right above the call to `assert.strictEqual()`.

Reply:
Thanks, @lpinca @Trott, for the comments. The message is on one line now.